turbinia/e2e/googleclouddisk.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+# This Turbinia e2e test will test the googleclouddisk functionality.
+# It will try to analyse the disk 'test-disk2' in zone 'us-central1-a'.
+
+
+TURBINIA_CLI="turbiniactl"
+MAIN_LOG="main.log"
+STATUS_LOG="status.log"
+STATS_LOG="stats.log"
+DETAIL_LOG="jobdetails.log"
+OUT_TGZ="jobresults.tgz"
+DISK="test-disk2"
+ZONE="us-central1-a"
+
+echo "Creating GCE test disk to use in e2e test"
+gcloud compute disks create $DISK --image=$DISK --zone=$ZONE
+
+echo "Executing googlecloudisk e2e test....this takes ~60 minutes!"
+$TURBINIA_CLI -L $MAIN_LOG -a -w googleclouddisk -d $DISK -z $ZONE &
+
+# Sleep here to make sure Turbinia can get started and we have a main.log file to tail.
+sleep 60
+
+# Parse turbiniactl output and wait for job ID.
+JOB_ID_LINE=`grep -m 1 "Creating request" <(tail -f $MAIN_LOG)`
```

Comment:
These should actually be Request IDs and not Job IDs.  (Each request can spawn many jobs, and each job can spawn 1 or more tasks, all three of which have their own IDs).

Reply:
FYI, you can actually specify the request ID on the command line from the client with `--request_id`, so you shouldn't need to parse it out of the logs.  You'd just need to make sure it's unique so that it doesn't conflict with other requests.  It looks like `uuidgen` is something available on my Ubuntu machine and could work for that.

Reply:
That's awesome that it's possible to specify the --request_id when creating a request. Will rewrite to use that!

Ack on the renaming of Job to Request terminology.